Libplanet.Explorer.Executable/Libplanet.Explorer.Executable.csproj

```diff
@@ -4,7 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <NoWarn>NU1701,SA1118</NoWarn>
+    <NoWarn>NU1701,SA1118,MEN003</NoWarn>
```

Comment:
We should not turn off this warning.  Which method does violate this?  [We can suppress a single source file or lines of a file.](https://docs.microsoft.com/en-us/visualstudio/code-quality/in-source-suppression-overview)

Reply:
The [Run method](https://github.com/planetarium/libplanet-explorer/pull/160/files#diff-4592f143a31d8fc9c98f043229eeb7041b611025ba1256630ca1772f722cfff9R40) in `Libplanet.Explorer.Executable/Program.cs` is currently around 240 lines but `MEN003` enforces a 120-line limit. I'm wasn't sure how to reduce the lines that much so I added this part. But I'll look into suppressing that method individually like you just suggested. Thanks!